tests/unit/gateways/test_block_api_gateway.py

```diff
@@ -0,0 +1,80 @@
+from unittest.mock import MagicMock
+
+from pytest import fixture
+
+from gateways.block_api_gateway import BlockApiGateway
+
+ELASTIC_SEARCH_BIGGEST_HEIGHT_SUCCESSFUL_RAW_RESPONSE = {
```

Comment:
I think we should have these big response bodies on the fixture folder, what do you think?

Reply:
Done in last review